### PR TITLE
Bug/5757

### PR DESCRIPTION
**Pull request by zz358840**
## Description

[redmine issue](https://redmine.kingkit.codes/issues/5757)

## Motivation and Context
led 閃爍 code 沒改到

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
-## [1.0.1] - 2022.09.12
+## [1.0.1] - 2022.09.16
```
Comment (GoGoPenguin) [inline CHANGELOG.md:9]: 就直接再進一版就好了，不用改原本的版本號

File: lib/webduino/led.py [hunk 2]
```diff
@@ -1,7 +1,16 @@
 from machine import Pin, PWM, Timer
 
 class LED:
+    def __new__(self,pin,strong=100,pwm=True):
+        if not hasattr(self, 'instances'):
+            self.instances = {}
+        if pin not in self.instances:
+            self.instances[pin] = super().__new__(self)
+        return self.instances[pin]
```
Comment (GoGoPenguin) [inline lib/webduino/led.py:9]: 可以解釋一下這邊為什麼要這麼做嗎？

Comment (zz358840) [reply lib/webduino/led.py:9]: 因為[這裡](https://github.com/webduinoio/pythonCode/pull/3/files#diff-0054566ab36c790f82cca9b127876e5111fb570398e9ffdf393bf62119c25d73R30)需要在呼叫一次，CamApp.py 裡面還有其他地方要用到 CamApp.led，所以做成 Singleton

File: CHANGELOG.md (hunk 1 above)
Comment (zz358840) [reply CHANGELOG.md:9]: @GoGoPenguin 
如果 led.py 還有要改我就整個改完一次 push，如果沒有我就只改這邊

**Pull request merged**